onyxia-api/src/main/java/fr/insee/onyxia/api/controller/api/mylab/MyLabController.java

```diff
@@ -240,6 +240,7 @@ private Collection<Object> publishApps(
         User user = userProvider.getUser(region);
         Map<String, Object> fusion = new HashMap<>();
         fusion.putAll((Map<String, Object>) requestDTO.getOptions());
+        fusion.putAll(region.getServices().getCustomValues());
```

Comment:
I am not sur of this one. 
I don't know why we should put that custom values here
@olevitt any thought ?

Reply:
Thanks for feedback!

> The mapping between configuration keys and helm package specific keys is done in web app based on x-onyxia specified in values.schema.json file.

Do I understand you correctly if you think that the values from `customValues` should be sent to web such that they can be edited there, @olevitt ?

I would say one approach to providing custom values in regions.json is to do the mapping in API, and not let the user be able to override these custom values.